package.json

```diff
@@ -31,6 +31,10 @@
           "roblourens.chat-agent-dalle.azureEndpoint": {
             "type": "string",
             "description": "The azure endpoint to make AI requests to, to generate images"
+          },
+          "roblourens.chat-agent-dalle.deploymentName": {
+            "type": "string",
+            "description": "The deployment name of the Dall-E model to use"
```

Comment:
What is the deployment name?

Reply:
Every deployment to Azure AI Studio has a deployment name.
Since version `1.0.0-beta.9`, the deployment name is required as part of the API:
https://github.com/Azure/azure-sdk-for-js/blob/%40azure/openai_1.0.0-beta.9/sdk/openai/openai/src/OpenAIClient.ts#L240

The old usage we had in this repo (using 1.0.0-beta.7) was creating a URL that resulted in a "Resource not found" error.